components/icons/Icons.tsx

```
@@ -0,0 +1,67 @@
+
+export const IconGithub = () => {
+    return (
+        <svg className="h-5 w-5" fill="currentColor" viewBox="0 0 24 24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)